descriptors/ModuleDescriptor-template.json

```diff
@@ -108,7 +108,7 @@
     },
     {
       "id": "circulation",
-      "version": "8.5",
+      "version": "9.0",
```

Comment:
The implementation version also needs to change.

Does this mean that a lot more coordinated pull requests are needed before any of these changes can be merged?

Reply:
Yes, mostly because of making item.copyNumbers to be a simple property. It was exposed in requests API.

Reply:
The implementation version has been changed.